src/license.js

```diff
@@ -1,5 +1 @@
-/*
-SPF
-(c) 2012-2015 Google, Inc.
-License: MIT
-*/
+/*! SPF | (c) 2012-2016 Google Inc. | License: MIT */
```

Comment:
It used to for uglify, but it looks like that's long gone:
https://github.com/mishoo/UglifyJS2#keeping-copyright-notices-or-other-comments

Removed in favor the standard comment